tests/test_widget.py

```diff
@@ -31,11 +31,21 @@ def test_open_widget(make_napari_viewer, bypass_load_screen, docking_area):
 
     # Check the widget behavior before and after loading the data
     if docking_area == "bottom" and bypass_load_screen is False:
```

Comment:
(Not related to this PR, but) do you know why the test is only for when the `docking_area` is at the bottom? Right docked widgets should also behave like this.

Reply:
I think because there used to be an:
```python
if docking_area!="bottom":
    self.bypass_load_screen = True
```
in the `__init__` that would have made some of the tests fail.... Sorry, it got a bit messy in the end.

Reply:
OK, testing for all dock positions now.